test-cases/tests/documentation/i04-accessing-rhoam.md

```diff
@@ -0,0 +1,28 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+    targets:
+      - 1.0
+tags:
+  - manual-selection
+---
+
+# I04 - Accessing rhoam
+
+## Description
+
+Procedure on getting access to RHOAM
+
+- Jira ref link:
```

Comment:
do we have jira link?

Reply:
no Jira link/it is essentially apart of the previous jira on adding rhoam